### PR TITLE
Add console log to doubles

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, this is only a thing that I consider useful when we run this project locally.

image

What do you think?

### CODE REVIEW

File: src/services/datahub-client.doubles.ts [hunk 1]
```diff
@@ -30,11 +30,13 @@ const fakeData = [
 
 export class HardcodedDatahubClient implements DatahubClient {
   public async getAccountDomains() {
+    console.log('getAccountDomains');
     await timeout(3000);
```
Comment (fcoronelmakingsense) [inline src/services/datahub-client.doubles.ts]: maybe for local 3seconds it's a lot of time. what do you think?

**Comment by fcoronelmakingsense** [comment]
Good! 👍

File: src/services/datahub-client.doubles.ts (hunk 1 above)
Comment (andresmoschini) [reply src/services/datahub-client.doubles.ts]: Agree, I am changing it. 

Thanks!

**Pull request merged**